app/views/layouts/base.html.slim

```diff
@@ -0,0 +1,24 @@
+- content_for :head
+  = csrf_meta_tags
+  = javascript_include_tag "application"
+  = stylesheet_link_tag "application", :media => "all"
+
+- content_for :content
+  = render "application/alerts"
+  = yield
+
+doctype html
+html lang="en-US"
+
+  head
+    meta charset="UTF-8"
+    title
+      = yield :title
```

Comment:
This is not working as expected. Do you have a recommendation for how to render title with a string?